.travis.yml

```diff
@@ -29,6 +29,8 @@ matrix:
           env: deps=high
         - php: 7.2
           env: deps=low
+        - php: 7.3
+          env: deps=low
```

Comment:
we should have only one `deps=low` job IMO (and as 7.3 is not stable yet, using it for the low deps is weird as they might trigger PHP deprecations or be incompatible btw).

Please remove the `deps=` config entirely here.